static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -605,8 +621,13 @@ export const getEpicTestsFromGoogleDoc = (): Promise<
                                 ','
                             ),
                             copy: {
-                                heading: replaceArticlesRead(
-                                    throwIfEmptyString('heading', row.heading)
+                                heading: replaceCountryName(
+                                    replaceArticlesRead(
+                                        throwIfEmptyString(
+                                            'heading',
```

Comment:
Confirming this is only ever expected to be in the heading, not in the body? That's more obvious to me regarding articles read but I could imagine the country name repeating in the body copy quite easily

Reply:
as discussed, I'll add this for the body as well. But it needs to be smarter to avoid scanning through lots of text for each test

Reply:
I've just pushed a commit that supports the country name in the body as well.
This change also adds a new boolean field, `hasCountryName`, to avoid having to scan through all the text looking for %%COUNTRY_NAME%%